### PR TITLE
add display of last error for failed submissions, fix Gutenberg block attribute types when downloading translations

### PR DESCRIPTION
**Pull request by vsolovei-smartling**

### CODE REVIEW

File: inc/Smartling/Helpers/GutenbergBlockHelper.php [hunk 1]
```diff
@@ -381,4 +382,38 @@ public function loadExternalDependencies()
...
+                                    break;
+                                case "number":
+                                    $attrs[$key] = (int)$attrs[$key];
+                                    break;
+                            }
+                        }
+                    }
+                }
+            }
+        }
```
Comment (PavelLoparev) [inline inc/Smartling/Helpers/GutenbergBlockHelper.php:415]: Tests are needed for all these branches

File: inc/Smartling/WP/WPAbstract.php [hunk 2]
```diff
@@ -357,7 +358,21 @@ public static function localeSelectionCheckboxBlock($namePrefix, $blog_id, $blog
...
+     * @param int $percentage
+     * @param string $lastError
+     * @param array $statusFlags
+     * @return string
+     */
+    public static function localeSelectionTranslationStatusBlock(
+        $statusText,
+        $statusColor,
+        $percentage,
+        $lastError = '',
```
Comment (PavelLoparev) [inline inc/Smartling/WP/WPAbstract.php:373]: I would add this new parameter with the default value to the end just to make sure new signature won't break any calls

File: inc/Smartling/Helpers/GutenbergBlockHelper.php (hunk 1 above)
Comment (vsolovei-smartling) [reply inc/Smartling/Helpers/GutenbergBlockHelper.php]: Done

File: inc/Smartling/WP/WPAbstract.php (hunk 2 above)
Comment (vsolovei-smartling) [reply inc/Smartling/WP/WPAbstract.php]: Done

**Pull request merged**